.github/CODEOWNERS

```diff
@@ -6124,7 +6124,7 @@
 /types/webpack-fail-plugin/                             @deevus
 /types/webpack-hot-client/                              @rynclark @ZSkycat
 /types/webpack-hot-middleware/                          @bumbleblym @icylace @chrisabrams
-/types/webpack-manifest-plugin/                         @r3nya
+/types/webpack-manifest-plugin/                         @r3nya @monsonjeremy
```

Comment:
This file is auto-generated. Please rollback this change.

Reply:
Thanks @armanio123, just fixed it!